x-pack/plugins/enterprise_search/common/types/workplace_search.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export interface IAccount {
+  id: string;
+  groups: string[];
+  isAdmin: boolean;
+  isCurated: boolean;
+  canCreatePersonalSources: boolean;
+  viewedOnboardingPage: boolean;
+}
+
+export interface IOrganization {
+  name: string;
+  defaultOrgName: string;
+}
```

Comment:
@scottybollinger FYI that I moved Workplace Search's IAccount/IOrganization types here (since both public/ and server/ files use it).

Also note that I deleted `IUser`, since we're no longer showing current user data (Kibana does that for us)